lib/matplotlib/pyplot.py

```diff
@@ -688,6 +689,18 @@ def savefig(*args, **kwargs):
     return res
 
 
+
+def encode_as(**kwargs):
+    """ Equivalent of savefig, but does not store to a file, but returns a bytestring
```

Comment:
This should probably be added to the boiler plate code to be auto-generated.  If not, then this should be

```python
return gcf().encode_as(**kwargs)
```

to avoid having two copies of the code around.